server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/NettyToResponseBody.scala

```diff
@@ -57,14 +70,46 @@ class NettyToResponseBody extends ToResponseBody[NettyResponse, NoStreams] {
     } yield (start, end + NettyToResponseBody.IncludingLastOffset)
 
     maybeRange match {
-      case Some((start, end)) => {
+      case Some((start, end)) =>
         val randomAccessFile = new RandomAccessFile(file, NettyToResponseBody.ReadOnlyAccessMode)
         new ChunkedFile(randomAccessFile, start, end - start, NettyToResponseBody.DefaultChunkSize)
-      }
       case None => new ChunkedFile(file)
     }
   }
 
+  private def convertToBuffs(bodyType: RawBodyType[_], part: Part[Any]): ByteBuf = {
+    bodyType match {
+      case RawBodyType.StringBody(_) =>
+        toPart(part.body, part.contentType, part.name, None)
+      case RawBodyType.ByteArrayBody =>
+        toPart(part.body, part.contentType, part.name, None)
+      case RawBodyType.ByteBufferBody =>
+        toPart(part.body, part.contentType, part.name, None)
+      case RawBodyType.InputStreamBody =>
+        toPart(part.body, part.contentType, part.name, None)
+      case RawBodyType.FileBody =>
+        val fileRange = part.body.asInstanceOf[FileRange]
+        toPart(Files.readString(fileRange.file.toPath), part.contentType, part.name, Some(fileRange.file.getName))
```

Comment:
Here the same, we can't read the file into memory

Reply:
As far as I understand we would need to introduce our own `ChunkedMultipart` data structure. It needs to be constructed by passing a list of `ByteBuf`s with small content and multipart parts' like headers and footers, and chunked streams and chunked files. And it needs to behave like `ChunkedFile` or `ChunkedStream` i.e. to implement `ChunkedInput<ByteBuf>`, a way to treat all the parts as something homogeneous, that could be read as a one whole. At the first glance, it seems to be quite tricky.